components/bsa/compressedbsafile.cpp

```diff
@@ -35,7 +35,16 @@
 
 #include <boost/iostreams/filtering_streambuf.hpp>
 #include <boost/iostreams/copy.hpp>
-#include <boost/iostreams/filter/zlib.hpp>
+
+#if defined(_WIN32)
```

Comment:
Is this supposed to be Win32 or MSVC? Do we want this with MinGW's GCC, too?

Reply:
More likely not, but I do not know if we support MinGW. Use the `_MSC_VER` instead, as in MyGUI's code?

Reply:
That's a reasonable solution. The gist of the MinGW situation is that we're trying to break things less frequently than we have MinGW users come along and fix things. If we had a bigger CI time budget on Windows, I'd like a CI job for it and then we could officially support it, but as-is, we don't have a regular developer who uses MinGW and our Windows devs are stretched thin enough already, so we don't have anyone to pick up the extra responsibility.